flowforge/input/BoundaryConditions.py

```diff
@@ -132,6 +132,42 @@ def _convertUnits(self, uc: UnitConverter) -> None:
             else:
                 raise Exception("Unknown enthalpy BC type: " + self.type_outlet)
 
+class VoidBC:
+    """Class for void fraction BC
+
+    Attributes
+    ----------
+    mdot : float
+        Mass float rate [kg/s]
+    """
+
+    def __init__(self, inlet: dict = None, outlet: dict = None):
+        assert inlet or outlet
+
+        self._type_inlet = None
+        self._val_inlet = None
```

Comment:
Is it possible to have this as an outlet?  If so, I'd go ahead and add it.

Reply:
Yes, added